README.md

```diff
@@ -103,7 +103,7 @@ retq
 
 ```cpp
 struct interface {
-  virtual ~iworld() noexcept = default;
+  virtual ~interface() noexcept = default;
```

Comment:
Good catch @mlimber.
Would you be able to add a link to a working example -> https://wandbox.org/permlink/so4nFDduIzx7m5sV
Thanks!